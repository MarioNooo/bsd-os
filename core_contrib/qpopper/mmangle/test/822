Date: Thu, 2 Jan 1992 12:21:18 -0800 (PST)
From: Laurence Lundblade <lgl@cac.washington.edu>
Sender: Laurence Lundblade <lgl@cac.washington.edu>
Reply-To: Laurence Lundblade <lgl@cac.washington.edu>
Subject: Pine source distribution
To: Bill Bathurst <bill@cpac.washington.edu>
yyy: extra header
xx: extra header
MIME-Version: 1.0
Content-type: "multipart"
 /
 (slkdfjslkfj) 
 mixed (sdlfkj_ ) ;
 boundary
 =
 "\A\b\C"


--AbC
content-type: application/xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxyyyyyyyyyyzzzzzzzzzzaaaaaaaaaabbbbbbbbbbcccccccccc
content-disposition: attachment; filename="a123456789b123456789c123456789d123456789e123456789f123456789g123456789h123456789i123456789j123456789.QCP"

xxxxxxxxxxx


--AbC
content-type: "application"/"fozboz"
content-disposition: attachment; filename="a123456789b123456789c123456789d123456789e123456789f123456789g123456789h123456789i123456789j123456789.QCP"

xxxxxxxxxxx

--AbC
content-type: application/foo
content-disposition: attachment; filename=xxx

xxxxxxxxxxx

--AbC
content-type: application(ffo)/(uuu")")foo
content-disposition: "attachment"; filename=xxx

--AbC
content-type: application"/""xxx"
content-disposition: "inline";filename="pp()pp"

xxxxxxxxxxx

--AbC
content-type: application"/""xxx"
content-disposition: attachment;filename=pp()pp

xxxxxxxxxxx

--AbC
content-type: (((()())))application (( ")))"
   )) "/"()()()
   (xxx"\"") "xyz"
content-disposition: attachment;filename=pp()pp

xxxxxxxxxxx

--AbC--





