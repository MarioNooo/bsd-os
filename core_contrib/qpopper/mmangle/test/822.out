Date: Thu, 2 Jan 1992 12:21:18 -0800 (PST)
From: Laurence Lundblade <lgl@cac.washington.edu>
Subject: Pine source distribution
To: Bill Bathurst <bill@cpac.washington.edu>


[ a123456789b123456789c123456789 ]
[ xxxxxxxxxxxxxxxxxxxx attachment omitted ]


[ a123456789b123456789c123456789 ]
[ fozboz attachment omitted ]


[ xxx ]
[ foo attachment omitted ]


[ xxx ]
[ foo attachment omitted ]


[ pp()pp ]
[ xxx part omitted ]


[ pp ]
[ xxx attachment omitted ]


[ pp ]
[ xyz attachment omitted ]

